ietf/settings.py

```diff
@@ -539,7 +539,7 @@ def skip_unreadable_post(record):
 IDTRACKER_BASE_URL = "https://datatracker.ietf.org"
 RFCDIFF_BASE_URL = "https://www.ietf.org/rfcdiff"
 IDNITS_BASE_URL = "https://author-tools.ietf.org/api/idnits"
-XML2RFC_BASE_URL = "https://xml2rfc.tools.ietf.org/experimental.html"
```

Comment:
yeah, it will start failing. @larseggert do you want us to copy the content somewhere else, or is it ok to just stop showing the floorplans for these older meetings? See https://tools.ietf.org/agenda/94/venue/ for an example...

Reply:
If it's easy to copy them somewhere, it;d be nice to keep them around

Reply:
I pulled what tools had and gave it to the secretariat - they'll work on back-filling FloorPlan objects.